app/models/pipeline_run.rb

```diff
@@ -411,6 +424,60 @@ def db_load_ercc_counts
     update(total_ercc_reads: total_ercc_reads)
   end
 
+  def db_load_insert_size_metrics
+    insert_size_metrics_s3_path = "#{host_filter_output_s3_path}/#{INSERT_SIZE_METRICS_OUTPUT_NAME}"
+    _stdout, _stderr, status = Open3.capture3("aws", "s3", "ls", insert_size_metrics_s3_path)
+    return unless status.exitstatus.zero?
+    insert_size_metrics_lines = Syscall.pipe_with_output(["aws", "s3", "cp", insert_size_metrics_s3_path, "-"])
+    tsv_lines = []
+    tsv_header_line = -1
+    insert_size_metrics_lines.split(/\r?\n/).each_with_index do |line, index|
```

Comment:
It may be possible to do a double parse but that won't be so clean. The file is a strange format. It is some text with two different csv's embedded. I will use `lines` though to clean it up.